homeassistant/components/media_player/sonos.py

```diff
@@ -351,11 +344,16 @@ def play_media(self, media_type, media_id, **kwargs):
             self._player.play_uri(media_id)
 
     @only_if_coordinator
-    def group_players(self):
+    def group_players(self, service):
```

Comment:
Why would you pass the service object to the media player? That's not something they should be aware off.